packages/paste-core/components/combobox/src/ComboboxListbox.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+
+export interface ComboboxListboxProps {
+  children: NonNullable<React.ReactNode>;
+}
+
+const ComboboxListbox: React.FC<ComboboxListboxProps> = ({children, ...props}) => {
+  return (
```

Comment:
I wonder if it’s worth leaving a TODO here as a note around needing to upgrade listbox positioning to use popperjs in the future?

Reply:
Agreed.